### PR TITLE
libcloud-plugin: replace treat_download_errors_as_warnings

### PR DESCRIPTION
**Pull request by franku**
- replace treat_download_errors_as_warnings by fail_on_download_error
- set fail_on_download_error to False by default because this is
  the default behaviour of bareos backups
- as a matter of fact by default download errors will be flagged
  as warnings and jobs will be marked "with warnings"

### CODE REVIEW

File: core/src/plugins/filed/python/libcloud/BareosFdPluginLibcloud.py [hunk 1]
```diff
@@ -350,20 +350,20 @@ def start_backup_file(self, savepkt):
...
                         % (self.current_backup_task["name"]),
                     )
-                    return bRC_Skip
+                    return bRC_Error
                 else:
                     jobmessage(
-                        M_ERROR,
-                        "File %s does not exist anymore"
+                        M_WARNING,
+                        "Skipped file %s because it does not exist anymore"
```
Comment (arogge) [inline core/src/plugins/filed/python/libcloud/BareosFdPluginLibcloud.py:363]: when `bopen()` in `SaveFile()` fails for whatever reason, it emits a jobmessage of type `M_NOTSAVED`:
```
Jmsg(jcr, M_NOTSAVED, 0, _("     Cannot open \"%s\": ERR=%s.\n"),                                                                                                                                                                            
           ff_pkt->fname, be.bstrerror());       
```
So maybe we should reconsider the message type here.

File: docs/manuals/source/TasksAndConcepts/Plugins.rst [hunk 2]
```diff
@@ -1636,10 +1636,11 @@ Optional Plugin Options:
 
 This option in the config file is optional:
 
-treat_download_errors_as_warnings
-   This parameter can be set to True to keep a job running if for some reason a file cannot
+fail_on_download_error
+   This parameter is by default False to keep a job running if for some reason a file cannot
    be downloaded from a bucket because it is either deleted or moved to another space during
```
Comment (arogge) [inline docs/manuals/source/TasksAndConcepts/Plugins.rst:1641]:
```suggestion
   When this option is enabled, any error during a file download will fail the backup job.
   By default a warning will be issued and the next file will be backed up.
```

File: docs/manuals/source/TasksAndConcepts/Plugins.rst [hunk 3]
```diff
@@ -1636,10 +1636,11 @@ Optional Plugin Options:
...
-   download. The default for this value is False.
+   download. In this case a Job will be marked with warnings as this is the default of Bareos.
+   It is recommended not to change this option.
```
Comment (arogge) [inline docs/manuals/source/TasksAndConcepts/Plugins.rst:1643]:
```suggestion
```

**Review by arogge — COMMENTED** [review]
We should take another look at the message-type for an object that cannot be loaded/opened.

File: core/src/plugins/filed/python/libcloud/BareosFdPluginLibcloud.py (hunk 1 above)
Comment (franku) [reply core/src/plugins/filed/python/libcloud/BareosFdPluginLibcloud.py:363]: This is the tested behaviour in the Python Plugin code. Code code is probably different.

**Pull request merged**